add task solution

solyaqw opened the pull request:
[DEMO LINK](https://solyaqw.github.io/layout_landing-page/)

SanyaBratashchuk reviewed (changes requested):
good start re-request review when finish whole landing

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
1. Fix gap between these blocks 
<img width="1118" alt="image" src="https://github.com/user-attachments/assets/b07a17c1-f758-4b11-b12d-e5a981e783eb">

2. Fix the placeholder text, check it on the design 
<img width="276" alt="image" src="https://github.com/user-attachments/assets/87fa0d91-dbc5-4558-8c31-c7c08b1a8be5">

3. Fix height for the textarea 
<img width="478" alt="image" src="https://github.com/user-attachments/assets/49e5de24-93b4-4c2c-913a-e4ecc567719f">

4. Fix font styles for the phone number 
<img width="211" alt="image" src="https://github.com/user-attachments/assets/6b58ab0a-32e3-4504-856b-454af0e0289d">

5. You need to hide the additional phone number on the mobile and tablet versions 
<img width="547" alt="image" src="https://github.com/user-attachments/assets/6a0eaece-2882-40fd-9886-5f6a92cc5ea1">

anastasiiavorobiova reviewed (changes requested):
Great work!
- <img width="799" alt="Screenshot 2024-10-22 at 15 08 04" src="https://github.com/user-attachments/assets/1e52874f-e831-46ed-8177-7ae8a4182037">
- Sections overlap on the scroll
- <img width="305" alt="Screenshot 2024-10-22 at 15 07 39" src="https://github.com/user-attachments/assets/713ee652-7326-48a7-8a6d-cfcb3e6c73dd">
- Check fonts here
- Consider removing default borders
- <img width="419" alt="Screenshot 2024-10-22 at 15 06 29" src="https://github.com/user-attachments/assets/ba877344-1731-48b8-8dfa-94beb8feaeb7">
- <img width="438" alt="Screenshot 2024-10-22 at 15 06 57" src="https://github.com/user-attachments/assets/7e22b55d-2d88-4d4f-85b0-2375b7f10e89">
- Check fonts here

solyaqw commented:
I added changes, corrected the menu__top when scrolling, corrected the font for the forms, but for some reason I did not see problems with fonts with prices, so if this problem still exists, please give some additional characteristics, in which cases does this font change, for example?

anastasiiavorobiova reviewed (changes requested):
Make sure to deploy your fixes

solyaqw commented:
![image](https://github.com/user-attachments/assets/32946b39-6dd0-491c-aa40-b1eb9221010f)

I've did a few minutes ago, before re-requesting

anastasiiavorobiova reviewed (changes requested):
Feel free to ask for help in the chat if you don't see your fixes on the deployed page

solyaqw commented:
![image](https://github.com/user-attachments/assets/955fc756-7bcd-4494-b95f-b1660f638074)
![image](https://github.com/user-attachments/assets/09f598ab-ce31-4f93-a33c-6c38157f6e11)
I fixed gaps, and now I wanna ask, if I really need to center prices on the same horizontal line? bcz on the figma design they're not centered

anastasiiavorobiova reviewed (approved):
Awesome!
